Feature 2489 update docs (#2717)
dtcenter/METplus: docs/Contributors_Guide/documentation.rst

See `Overleaf's Guide - Matrices <https://www.overleaf.com/learn/latex/Matrices>`_ for more information.

Literal Include
---------------

> This feature is used often in the Release Guide.  But there are other places it can also be used.

Reviewer: ```suggestion
This feature is used often in the Release Guide, but there are other places it could be useful.
```
I am modifying this from the original text we had.  Thank you for putting it in as-is and apologies for the last minute change.

Using a literal include ensures only having to update the documentation in one place.

> It is also important to use a relative path (e.g. "../../") to get to the correct 

Reviewer: ```suggestion
It is important to use a relative path (i.e. "../../") to get to the correct 
```
Again, I'm modifying the original text.

directory of the file being referenced in the literal include.  This will keep the 
file linking to the correct version and branch.  

As an example, refer to the METplus User's Guide.  See this 
`line <https://github.com/dtcenter/METplus/blob/46f705fc2c1b861e3aa6314d030eb9b4c382d0b3/docs/use_cases/met_tool_wrapper/PointStat/PointStat.py#L55>`_ 

> in the **PointStat.py** which uses RST's :code:`.. literalinclude::` 

Reviewer: ```suggestion
in the PointStat Wrapper Use Case documentation, which uses RST's :code:`.. literalinclude::` 
```
Again, I'm modifying the original text.

> and renders on the web in what appears to be a code block in this 

Reviewer: ```suggestion
and renders the text of the referenced file in this 
```
Again, I'm modifying the original text.

`literal blocks <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks>`_
for more information.

Substitution References
-----------------------

Substitution references are replacements in the text as defined in the :code:`rst_epilog` 
in the **docs/conf.py** file.  For example, instead of hard coding the MET version in 
the METplus documentation, a variable :code:`met_version` can be set to a value 

> e.g. 12.0.0) in the **docs/conf.py** file in the following way:

Reviewer: ```suggestion
(e.g. 12.0.0) in the **docs/conf.py** file in the following way:
```
Adding in missing parenthesis
